pubber/src/main/java/daq/pubber/Pubber.java

```diff
@@ -859,19 +859,22 @@ private void configHandler(Config config) {
 
   private void processConfigUpdate(Config config) {
     final int actualInterval;
+    final int useInterval;
```

Comment:
can declare at point of use in this case since there is no branch involved.

Reply:
Changed